Expired/Views/ProductListView.swift

```diff
@@ -17,6 +17,27 @@ struct ProductListView: View {
 
 
     var body: some View {
+        TabView {
+            // 1st Tab
+            listView
+//            .font(.system(size: 12, weight: .bold, design: .rounded))
+            .tabItem {
+                Image(systemName: "house.fill")
+                Text("Home")
+            }
+            
+            //2nd Tab
+            SettingView()
+//            .font(.system(size: 12, weight: .bold, design: .rounded))
+            .tabItem {
+                Image(systemName: "gear.circle.fill")
+                Text("Settings")
+            }
```

Comment:
```suggestion
            listView
                .tabItem {
                    Image(systemName: "house.fill")
                    Text("Home")
                }

            // 2nd Tab
            SettingView()
                .tabItem {
                    Image(systemName: "gear.circle.fill")
                    Text("Settings")
                }
```

- Indentation of modifiers
- Delete commented code to keep the code clean
- Add space between "//" and "2nd Tab"